doc/datasets/rcv1.rst

```diff
@@ -32,10 +32,10 @@ The target values are stored in a scipy CSR sparse matrix, with 804414 samples a
 Each sample can be identified by its ID, ranging (with gaps) from 2286 to 810596::
 
     >>> rcv1.sample_id[:3]
-    array([2286, 2287, 2288], dtype=int32)
+    array([2286, 2287, 2288], dtype=uint32)
```

Comment:
that's weird as travis was not complaining. I would revert as it's unrelated to this PR